pyvista/core/utilities/arrays.py

```diff
@@ -248,12 +248,19 @@ def convert_array(
 def convert_array(
     arr: npt.ArrayLike, name: str | None = ..., deep: bool = ..., array_type: int | None = None
 ) -> _vtk.vtkAbstractArray: ...
+@overload
+def convert_array(
+    arr: None,
+    name: str | None = None,
+    deep: bool = False,
+    array_type: int | None = None,
```

Comment:
The overloads are tricky, I _think_ using ellipses here will make the overload work in cases like `deep=True` or something like that.

```suggestion
    name: str | None = ...,
    deep: bool = ...,
    array_type: int | None = ...,
```